### DISCUSSION TITLE
Loading/ Parsing SVG files with rspack

### DISCUSSION THREAD

**Discussion by pushkargupta9807** [web-infra-dev/rspack#3027] [General]
Hi, 
from the webpack world we have a loader svg-react-loader which is used as below in the file importing  the svg asset and creating react component. 

Examle:
var TaskComplete = require('-!@svg-react-loader?name=Unlock!../../../lib/Assets/TaskComplete.svg');;
......
 icons.TaskComplete = React.createElement(TaskComplete, null);
 
 
 
 I tried and I am not sure if svg-react-loader is compatible with rspack yet. With the compatible  @svgr/webpack I am also not sure , what changes would I have to make for the above code to work. 
 
 Any ideas, suggestions would be greatly appreciated. Thanks!

**Comment by hardfist** [comment]
svg-react-loader is supported, you can see a example here https://github.com/web-infra-dev/rspack/pull/3030
